com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -439,21 +439,15 @@ void UpgradeResourcesIfNeeded()
             m_GlobalSettings.EnsureEditorResources(forceReload: true);
 
             bool requiresRayTracingResources = false;
-            if (GatherRayTracingSupport(asset.currentPlatformRenderPipelineSettings))
+            // Make sure to include ray-tracing resources if at least one of the quality levels needs it
+            int qualityLevelCount = QualitySettings.names.Length;
+            for (int i = 0; i < qualityLevelCount && !requiresRayTracingResources; ++i)
             {
-                requiresRayTracingResources = true;
-            }
-            // Also make sure to include ray-tracing resources if at least one of the quality levels needs it
-            else if (rayTracingSupportedBySystem)
-            {
-                int qualityLevelCount = QualitySettings.names.Length;
-                for (int i = 0; i < qualityLevelCount && !requiresRayTracingResources; ++i)
-                {
-                    var hdrpAsset = QualitySettings.GetRenderPipelineAssetAt(i) as HDRenderPipelineAsset;
-                    if (hdrpAsset != null && hdrpAsset.currentPlatformRenderPipelineSettings.supportRayTracing)
-                        requiresRayTracingResources = true;
-                }
+                var hdrpAsset = QualitySettings.GetRenderPipelineAssetAt(i) as HDRenderPipelineAsset;
+                if (hdrpAsset != null && hdrpAsset.currentPlatformRenderPipelineSettings.supportRayTracing)
```

Comment:
same check if we need to check hardware compatibility too

Reply:
We must not check hardware as a build could be done on a DX12 compatible computer which don't have RTX card.